pkg/server/server.go

```diff
@@ -314,27 +278,29 @@ func (server *Server) rotateSigningCert() error {
 	return err
 }
 
-func getRegistrationEndpoints(registrationSvc registration.Service) registration.Endpoints {
-	return registration.Endpoints{
-		CreateEntryEndpoint:           registration.MakeCreateEntryEndpoint(registrationSvc),
-		DeleteEntryEndpoint:           registration.MakeDeleteEntryEndpoint(registrationSvc),
-		FetchEntryEndpoint:            registration.MakeFetchEntryEndpoint(registrationSvc),
-		UpdateEntryEndpoint:           registration.MakeUpdateEntryEndpoint(registrationSvc),
-		ListByParentIDEndpoint:        registration.MakeListByParentIDEndpoint(registrationSvc),
-		ListBySelectorEndpoint:        registration.MakeListBySelectorEndpoint(registrationSvc),
-		ListBySpiffeIDEndpoint:        registration.MakeListBySpiffeIDEndpoint(registrationSvc),
-		CreateFederatedBundleEndpoint: registration.MakeCreateFederatedBundleEndpoint(registrationSvc),
-		ListFederatedBundlesEndpoint:  registration.MakeListFederatedBundlesEndpoint(registrationSvc),
-		UpdateFederatedBundleEndpoint: registration.MakeUpdateFederatedBundleEndpoint(registrationSvc),
-		DeleteFederatedBundleEndpoint: registration.MakeDeleteFederatedBundleEndpoint(registrationSvc),
+func (server *Server) getGRPCServer() (*grpc.Server, error) {
+	// TODO: Fix me after server refactor
+	crtRes, err := server.dependencies.ServerCAImpl.FetchCertificate(&ca.FetchCertificateRequest{})
```

Comment:
Not related to the PR but wonder if we can get rid of dependencies.. right now we have init_dependencies and init_plugins, maybe we could combine them both

Reply:
After your previous comment on server.go I noticed the dependencies. I'm about to push changes that remove them.